src/ralph/accounts/admin.py

```diff
@@ -117,20 +117,41 @@ def user_licence(self, item):
             return []
 
     def buyout_ticket(self, item):
-        if not item.model.category.show_buyout_date:
-            return ''
+        if not item.custom_fields_as_dict.Purpose:
+            if not item.model.category.show_buyout_date:
+                return ''
+            else:
```

Comment:
Remove else, move the whole block to the end of this method to avoid code duplication.